### DISCUSSION TITLE
keyball39 マウスポインターが勝手に動いてしまう

### DISCUSSION THREAD

**Discussion by jun3366** [Yowkees/keyball#164] [Q&A]
マウスポインターが勝手に動いてしまいます。
現在の症状がトラックボールを動かすとポインターが上下に小さく振動しながら、画面上や画面下に勝手に動いてしまいます。
ただ、ずっと動いているわけでもなく、トラックボールを触らず10秒前後放置するとポインターの振動が止まることもあります。

また、トラックボールを外すとポインターが大きく振動し、制御ができない状況になります。
トラックボールを付けると上記のような挙動を示します。

上記情報でわかる対策案ございましたらご教授お願い致します。

**Comment by Yowkees** [comment]
マウスポインタが勝手に動く症状は、半田付け不良により起こります。
センサーIC、L字7Pコンスルーとその近くに4つのジャンパー、ProMicro本体の半田付けを見直してください。

半田が基板上の銀パッドにうまく吸着しているかが重要で、ボールのような形をしている半田付けは山形になるようにやり直してみてください。

お手数ですがご検討よろしくお願いします。

**Reply by jun3366** [reply]
はんだ付を再度行ったら正常に作動しました。
現在は問題なく使えておりアドバイス感謝いたします。

**Reply by shunji-mik** [reply]
私も同様の症状が出たのですが、再度、センサーICとL字コンスルーの半田付けを行った所、症状が解消しました。

**Comment by Yowkees** [comment]
半田付けの修正ありがとうございました。
無事に動くようになって良かったです。
ぜひ末永くご愛用いただければ幸いです。
